js/id/ui/alert.js

```diff
@@ -44,10 +44,8 @@ iD.ui.Alert = function(message,type,stack) {
 
     var prettyString = function(obj) {
         var finalString = '';
+        obj.commandDetail =  JSON.stringify(obj.commandDetail, null, '<br>');
```

Comment:
This change seems to be the culprit that breaks tests.  Not sure why, but I guess put the line back in the for loop.